common/notification/actions.ts

```diff
@@ -618,6 +618,10 @@ const _notificationActions = {
     // ACHIEVEMENT TEST ACTIONS
 
     /* ONBOARDING */
+    reward_issued: {
+        description: 'Reward issued',
+        sampleContext: {},
```

Comment:
Maybe we have to define a context here. I would wait for more informations here.
I could imagine that we could use the `achievement name` and `achievedAt`